packages/form-js-viewer/src/render/components/form-fields/ExpressionField.js

```diff
@@ -1,12 +1,13 @@
 import { useCallback, useEffect } from 'preact/hooks';
-import { useExpressionEvaluation, useDeepCompareMemoize, useService, useEffectOnChange } from '../../hooks';
```

Comment:
`useEffectOnChange` is not used anymore, let's delete it

Reply:
Nicely spotted, removed it :)